yarn.lock

```diff
@@ -1282,6 +1282,32 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@fluentui-react-native/framework@0.6.6":
+  version "0.6.6"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/framework/-/framework-0.6.6.tgz#5f5c37ccf2dad07248fa3c7f3b5d8497ec08bbbd"
+  integrity sha512-OlVj1QraznydLUdYtNrPXhYN0a8YldwOMgRoyTXS4HG8K4a2E2BhuzBb89QLjIVtNV0KB6bZbE5SUCJpOJHOaA==
+  dependencies:
+    "@fluentui-react-native/composition" ">=0.6.12 <1.0.0"
+    "@fluentui-react-native/default-theme" ">=0.8.11 <1.0.0"
+    "@fluentui-react-native/immutable-merge" "^1.1.4"
+    "@fluentui-react-native/memo-cache" "^1.1.5"
+    "@fluentui-react-native/merge-props" ">=0.3.5 <1.0.0"
+    "@fluentui-react-native/theme-types" ">=0.12.4 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.10.3 <1.0.0"
+    "@fluentui-react-native/use-slot" ">=0.1.7 <1.0.0"
+    "@fluentui-react-native/use-slots" ">=0.5.9 <1.0.0"
+    "@fluentui-react-native/use-styling" ">=0.7.2 <1.0.0"
+    "@fluentui-react-native/use-tokens" ">=0.1.7 <1.0.0"
+    tslib "^1.13.0"
+
+"@fluentui-react-native/theme-tokens@^0.9.2":
+  version "0.9.2"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/theme-tokens/-/theme-tokens-0.9.2.tgz#09ef091c878338f8a80b0b00894d94a1abe9bce7"
+  integrity sha512-c95ntUKM6y6N1pmQdP7uSSYxbeiwAQJYvxapkcK7+WHFedVi5iIZCo7qC/q2zF5+H5xV76DaCJCzDYl9jXk75A==
+  dependencies:
+    "@fluentui-react-native/theme-types" ">=0.12.4 <1.0.0"
+    assert-never "^1.2.1"
+
```

Comment:
The presence of these (and other yarn.lock changes) implies your new package is using some version of "@fluentui-react-native/framework" that is out of sync with the rest of the monorepo, so instead of referring to the that dependency from your local clone of the monorepo, it's pulling it from NPM (I think). Might want to double check that.